porch/pkg/engine/engine.go

```diff
@@ -328,14 +330,28 @@ func (cad *cadEngine) UpdatePackageRevision(ctx context.Context, repositoryObj *
 		mutations = append(mutations, mutation)
 	}
 
-	// Re-render if we are making changes.
-	mutations = cad.conditionalAddRender(mutations)
-
 	draft, err := repo.UpdatePackageRevision(ctx, oldPackage)
 	if err != nil {
 		return nil, err
 	}
 
+	// If any of the fields in the API that are projections from the Kptfile
```

Comment:
It's fine to try this, but I suspect we won't be able to maintain this in the Kptfile going forwards.

Reply:
You mean that projecting fields from the Kptfile into the API here? If so, why? I had some concerns about it, but it actually seems to work a bit better than I feared.